.pyup.yml

```diff
@@ -1,8 +1,6 @@
-schedule: "every day"
+schedule: "every week on monday"
 search: False
-update: insecure
 requirements:
   - tests/e2e/Pipfile:
-      update: security
       pin: True
```

Comment:
The default is True, so we could remove this line (and the colon at the end of the line above).

Reply:
Fixed; thanks!